src/develop/ui/patterns/mobile/gallery.md

```diff
@@ -3,60 +3,49 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# Gallery Pattern
+# Gallery 
 
-The Gallery pattern
-displays content (such as cards) in a specific set of columns, configurable
-per device type and orientation. Use this pattern to display a List of
-elements side by side, with a different number of items per row on different
-devices.
+You can use the Gallery UI Pattern to display content (such as cards) in a specific set of columns, configurable per device type and orientation. 
 
-Here's a preview in Service Studio:
+![](<images/gallery-image-7.png>)
 
-![](images/Gallery_preview.png)
 
-## How to Use the Gallery Pattern
+## How to use the Gallery UI Pattern
 
-1\. Drag a list or static content.
+1. In Service Studio, in the Toolbox, search for `Gallery`. 
 
-2\. Set the number of items in Input Parameters.
+    The Gallery widget is displayed.
 
-### Using Animations Inside the Gallery
+    ![](<images/gallery-image-8.png>)
 
-1\. Use the **Animate** block as the first element inside the list.
+1. From the Toolbox, drag the Gallery widget onto your application’s screen.
 
-![](images/Gallery_animate.png)
 
-2\. Place your content inside the b block.
+1.  Drag a list or static content into the Content placeholder.
```

Comment:
Drag a List Records (LINK) widget or static content into the Content placeholder